auth0/provider.go

```diff
@@ -44,6 +49,7 @@ func configure(data *schema.ResourceData) (interface{}, error) {
 	domain := data.Get("domain").(string)
 	id := data.Get("client_id").(string)
 	secret := data.Get("client_secret").(string)
+	debug := data.Get("debug").(bool)
```

Comment:
I believe this is why the build fails, its defined as a string?

Reply:
I know, sorry, this is just a draft yet ;)

Reply:
No worries :)